CHANGES.rst

```diff
@@ -1,6 +1,21 @@
 Changelog
 =========
 
+Version 0.2.0 [2020-09-16]
+--------------------------
+
+Features
+~~~~~~~~
+
+- Support for Django 3.1
+- Feature to silence notifications for specific objects
+  temporarily or permanently
```

Comment:
can you please link https://github.com/openwisp/openwisp-notifications#silencing-notifications-for-specific-objects-temporarily-or-permanently ?